### PR TITLE
SRVKP-6184 - Add Running PipelineRuns metric for Pipeline and Namespace.

### PR DESCRIPTION
**Pull request by pramodbindal**
# Changes
Updated metric Running PipelineRuns to include namespace and pipeline

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) if any changes are user facing, including updates to minimum requirements e.g. Kubernetes version bumps
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [ ] [pre-commit](https://github.com/tektoncd/pipeline/blob/main/DEVELOPMENT.md#install-tools) Passed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including functionality, content, code)
- [x] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings). See some examples of [good release notes](https://github.com/tektoncd/community/blob/main/standards.md#good-release-notes).
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by pramodbindal** [comment]
++ @vdemeester  - please have a look

File: pkg/pipelinerunmetrics/metrics.go [hunk 1]
```diff
@@ -213,6 +213,7 @@ func viewRegister(cfg *config.Metrics) error {
 		Description: runningPRs.Description(),
 		Measure:     runningPRs,
 		Aggregation: view.LastValue(),
+		TagKeys:     append([]tag.Key{namespaceTag}, prunTag...),
```
Comment (khrm) [inline pkg/pipelinerunmetrics/metrics.go:216]: This should be configurable based on some key in config map.

**Review by khrm — COMMENTED** [review]
/ok-to-test

**Comment by pramodbindal** [comment]
label kind/feature

File: pkg/pipelinerunmetrics/metrics.go (hunk 1 above)
Comment (pramodbindal) [reply pkg/pipelinerunmetrics/metrics.go:216]: I believe we should always add the namespace tag.
currently prunTag is being used at two places and both with namespaces. what about adding the namespace in pruntag only ?

Comment (khrm) [reply pkg/pipelinerunmetrics/metrics.go:216]: Please see this issue: https://github.com/tektoncd/pipeline/issues/3171

Comment (khrm) [reply pkg/pipelinerunmetrics/metrics.go:216]: Also this: https://github.com/knative/pkg/pull/1464

Comment (khrm) [reply pkg/pipelinerunmetrics/metrics.go:216]: https://github.com/knative/pkg/pull/2931

**Review by vdemeester — APPROVED** [review]

**Comment by khrm** [comment]
Please squash all commits and rebase with the latest changes in the main.

**Pull request closed without merging**